src/cache_memoize/__init__.py

```diff
@@ -102,8 +102,9 @@ def _default_make_cache_key(*args, **kwargs):
                 [quote(str(x)) for x in args_rewrite(*args)]
                 + [quote("{}={}".format(k, v)) for k, v in kwargs.items()]
             )
+            prefix = prefix or ".".join((__name__, func.__qualname__))
             return hashlib.md5(
-                force_bytes("cache_memoize" + (prefix or func.__qualname__) + cache_key)
```

Comment:
Functions with the same name from different modules share a common cache (because `__qualname__` does not contain the module name, see [docs](https://docs.python.org/3/glossary.html#term-qualified-name))